sample/build.gradle

```diff
@@ -33,27 +28,26 @@ android {
 dependencies {
 
     // Android
-    compile 'com.android.support:appcompat-v7:23.1.1'
-    compile 'com.android.support:recyclerview-v7:23.1.1'
-    compile 'com.android.support:cardview-v7:23.1.1'
+    implementation 'com.android.support:appcompat-v7:28.0.0'
+    implementation 'com.android.support:recyclerview-v7:28.0.0'
+    implementation 'com.android.support:cardview-v7:28.0.0'
```

Comment:
I think these should be AndroidX dependencies since you are using AndroidX xml tags in the layout.

Reply:
👍 good catch